Helper/Invoice.php

```diff
@@ -249,14 +261,19 @@ public function handleCaptureWebhook(Order $order, Notification $notification):
     {
         $invoiceFactory = $this->adyenInvoiceFactory->create();
         $adyenInvoice = $this->adyenInvoiceResourceModel->getAdyenInvoiceByCaptureWebhook($order, $notification);
+        $fullAmountCaptured = $this->adyenDataHelper->originalAmount($notification->getAmountValue(), $notification->getAmountCurrency()) >= $order->getBaseGrandTotal();
 
-        if (is_null($adyenInvoice)) {
-            throw new \Exception(sprintf(
-                'Unable to find adyen_invoice linked to original reference %s, psp reference %s, and order %s',
-                $notification->getOriginalReference(),
-                $notification->getPspreference(),
-                $order->getIncrementId()
-            ));
+        if (is_null($adyenInvoice) && $order->canInvoice()) {
```

Comment:
If `$adyenInvoice` is `null` and we cannot invoice (`$order->canInvoice()` is `false`), can we still `throw` the exception?

Reply:
👌